docs/cli-reference/start.md

```diff
@@ -15,27 +15,39 @@ gpustack start [OPTIONS]
-| `--ray-node-manager-port` value           | `40098`                                | Port of Ray node manager. Used when Ray is enabled.                                                                                                                                                                                                                                                   |
-| `--ray-object-manager-port` value         | `40099`                                | Port of Ray object manager. Used when Ray is enabled.                                                                                                                                                                                                                                                 |
-| `--ray-runtime-env-agent-port` value      | `40100`                                | Port of Ray runtime env agent. Used when Ray is enabled.                                                                                                                                                                                                                                              |
-| `--ray-dashboard-agent-grpc-port` value   | `40101`                                | Port for Ray dashboard agent gPRC listen. Used when Ray is enabled.                                                                                                                                                                                                                                   |
-| `--ray-dashboard-agent-listen-port` value | `52365`                                | Port for Ray dashboard agent HTTP listen. Used when Ray is enabled.                                                                                                                                                                                                                                   |
-| `--ray-metrics-export-port` value         | `40103`                                | Port for Ray metrics export. Used when Ray is enabled.                                                                                                                                                                                                                                                |
+| <div style="width:180px">Flag</div>         | <div style="width:100px">Default</div> | Description                                                                                                                           |
+| ------------------------------------------- | -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| `--advertise-address` value                 | (empty)                                | The address to expose for external access. Auto-detected by default.                                                                  |
```

Comment:
Can we make the help text clearer? e.g.,
```
The IP address or hostname to expose for external access.
If not set, the system will auto-detect a suitable local IP address.
```

Reply:
Updated